model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/organization/CachedOrganizationIds.java

```diff
@@ -28,21 +28,28 @@
 
 public class CachedOrganizationIds extends AbstractRevisioned implements InRealm {
 
-    private final RealmModel realm;
-    private final Set<String> orgIds = new HashSet<>();
+    private final String realmId;
+    private final List<String> orgIds;
 
-    public CachedOrganizationIds(Long revision, String id, RealmModel realm, Stream<OrganizationModel> organizations) {
+    public CachedOrganizationIds(Long revision, String id, RealmModel realm, OrganizationModel model) {
         super(revision, id);
-        this.realm = realm;
-        organizations.map(OrganizationModel::getId).forEach(orgIds::add);
+        this.realmId = realm.getId();
+        orgIds = List.of(model.getId());
```

Comment:
The reason why it is a set is because we don't want duplicates even though very unlikely to happen. Or is there any other reason to change to a list?

Reply:
This change was made by @pruivo. He mentioned lists are more effective. We can change to set and then to list to make sure there are not duplicates.

Reply:
No problem. Let's keep as is.

Reply:
I changed it to convert to set and only then to list to make sure there are no duplicates. @pruivo does it make sense to you?

Reply:
`List` is more memory efficient than `Set`; we are not invoking `contains()` and only iterating. We can use  `distinct()` in the stream if you want to remove duplicates. 
I'm ok to use `Set` if you prefer :+1: